public/class-ed-solr-public.php

```diff
@@ -39,8 +39,8 @@ class Ed_Solr_Public {
 	 * Initialize the class and set its properties.
```

Comment:
Another, remove description form the top.

Reply:
No, think that is fine. Just describes what the constructor is doing

Reply:
I'm sorry Richard, I meant at line 1, not where I put the content in. It is difficult to do it on github.

Reply:
done